### PR TITLE
general fixes to sdk tests

### PR DESCRIPTION
**Pull request by makk94**

### CODE REVIEW

**Comment by mireaulf** [comment]
Why the change?

**Review by privateducky — COMMENTED** [review]
don't add the package-lock.json file to this PR

File: javascript/sdk/package.json [hunk 1]
```diff
@@ -21,8 +21,10 @@
     "prepublishOnly": "yarn build",
     "test": "vitest"
   },
+  "engines": {
+    "node": ">=18"
+  },
   "devDependencies": {
-    "@theprelude/sdk": "^0.9.58",
     "@types/chai": "^4.3.4",
     "@types/mocha": "^10.0.1",
```
Comment (kenrick) [inline javascript/sdk/package.json:29]:
```suggestion
```

**Review by mireaulf — APPROVED** [review]

**Review by kenrick — CHANGES_REQUESTED** [review]
can remove these

**Comment by mireaulf** [comment]
> ah, the update account is updating it to FROZEN which means don't run tests. and we need to run a test to get some activity in the detect_controller_tests

File: javascript/sdk/package.json (hunk 1 above)
Comment (makk94) [reply javascript/sdk/package.json]: done

**Pull request merged**